EmailValidator/Validation/DNSCheckValidation.php

```diff
@@ -18,6 +18,13 @@ class DNSCheckValidation implements EmailValidation
      * @var InvalidEmail
      */
     private $error;
+    
+    public function __construct()
+    {
+        if (!extension_loaded('intl')) {
+            throw new \LogicException(sprintf('The %s class requires the Intl extension.', __CLASS__));
```

Comment:
Instead of a generic message, what about something like `%s feature requires the Intl extension.` ?

Reply:
I just copied the same message from the Spoof validator :)